bin/build.sh

```diff
@@ -451,4 +454,17 @@ then
   puts "Regression tests complete"
 fi
 
+if [ ${INSTALL_OBJECTIVEC} -eq 1 ]
+then
+  puts "Installing GNU Objective-C for linux"
+
+  sudo apt-get -y install gobjc gnustep gnustep-make gnustep-common gnustep-devel
+  # The version numbers here will have to change by OS.
+  sudo ln -s /usr/lib/gcc/x86_64-linux-gnu/4.8/include/objc /usr/local/include/objc
+  echo ". /usr/share/GNUstep/Makefiles/GNUstep.sh" >> ~/.bashrc
```

Comment:
If this step is required, I think it's better we put `. /usr/share/GNUstep/Makefiles/GNUstep.sh` into `smack.environment`.

Reply:
Is smack.environment a generated file? I can't find it in the repo

Reply:
Yes. It's generated by `build.sh`. See this: https://github.com/smackers/smack/blob/develop/bin/build.sh#L48.